src/components/side_panel/pivot/pivot_layout_configurator/pivot_dimension/pivot_dimension.xml

```diff
@@ -4,7 +4,24 @@
       class="border py-1 px-2 d-flex flex-column shadow-sm pivot-dimension"
       t-att-class="{'bg-danger': !props.dimension.isValid}">
       <div class="d-flex flex-row justify-content-between align-items-center">
-        <span class="fw-bold" t-esc="props.dimension.displayName"/>
+        <div class="d-flex align-items-center">
+          <t t-if="state.isUpdate">
```

Comment:
what about being always in edit mode ? Just like in odoo. It has worked out pretty well.
It's less code and less friction for the user